Regression/Checksum/benchmarks_json/averaged_galilean_2d_psatd.json

```diff
@@ -2,32 +2,32 @@
   "electrons": {
     "particle_cpu": 0.0,
     "particle_id": 2147516416.0,
-    "particle_momentum_x": 7.840028236162465e-21,
-    "particle_momentum_y": 3.765146435729977e-21,
-    "particle_momentum_z": 1.7807679253921786e-16,
-    "particle_position_x": 405588.5591398962,
-    "particle_position_y": 20127109.071436103,
+    "particle_momentum_x": 0.0000000000000000000025448898780350552577,
+    "particle_momentum_y": 0.0000000000000000000037848971271928787262,
+    "particle_momentum_z": 0.0000000000000001780767630908322081725904,
+    "particle_position_x": 405588.5890661150915548205375671386718750000000,
+    "particle_position_y": 20127109.1190979480743408203125000000000000000000,
     "particle_weight": 6.917460794691972e+17
   },
   "ions": {
     "particle_cpu": 0.0,
     "particle_id": 6442483712.0,
-    "particle_momentum_x": 2.6093404228219635e-18,
-    "particle_momentum_y": 2.6179500432515532e-18,
+    "particle_momentum_x": 0.0000000000000000026093726328748384946073,
+    "particle_momentum_y": 0.0000000000000000026179524810745284785374,
     "particle_momentum_z": 3.269760998813484e-13,
     "particle_position_x": 405588.43859727494,
     "particle_position_y": 20127109.11750953,
     "particle_weight": 6.917460794691972e+17
   },
   "lev=0": {
-    "Bx": 0.0037913173274310057,
-    "By": 0.0019410303151043715,
-    "Bz": 0.010731882944058382,
-    "Ex": 471961.34461594024,
-    "Ey": 3576283.0065883463,
-    "Ez": 309019.28003672883,
-    "jx": 221.62156557908702,
-    "jy": 993.3580780378838,
-    "jz": 4650.141880076047
+    "Bx": 0.0039214717277128371963890174356492934749,
+    "By": 0.0019317334284915893636430661572944700310,
+    "Bz": 0.0110672482515600210939421543798744096421,
+    "Ex": 490051.7683783262036740779876708984375000000000,
+    "Ey": 3567406.0230688336305320262908935546875000000000,
+    "Ez": 281040.1780148115358315408229827880859375000000,
+    "jx": 219.6458000289931078441441059112548828125000,
+    "jy": 985.4589204205597070540534332394599914550781,
+    "jz": 4147.9419588566606762469746172428131103515625
```

Comment:
I expected this PR to only affect machine precision errors (at least with the default number of guard cells and without instability), but the changes in the benchmarks of this test are rather large. Is this expected or is there something to understand here?

Reply:
I would not expect large changes on this specific test, especially because it seems to me that the test runs on 1 MPI process
https://github.com/ECP-WarpX/WarpX/blob/b575206664836ba5df3e1cf1976148d9810c0a65/Regression/WarpX-tests.ini#L1764-L1774
and without domain decomposition
https://github.com/ECP-WarpX/WarpX/blob/b575206664836ba5df3e1cf1976148d9810c0a65/Examples/Tests/averaged_galilean/inputs_avg_2d#L6-L8
I don't understand these changes in the benchmark at this point.

Reply:
I think this can be because the domain is periodic so there is some duplication of nodal points at the edges of the domain.

Also I've looked at the test and it appears to be unstable (field energy starts rising exponentially at the end). I'm not sure if this is expected but this can explain why small changes in the code can result in significant changes in the benchmarks.